drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -286,10 +286,6 @@ msgid ""
 msgstr ""
 
 #: templates/html/js_templates.html
-msgid "Zones"
```

Comment:
@mtyaka @itsjeyd @haikuginger `msgfmt` considers multiple occurence of same `msgid` as a fatal error. So, here's the question - have you actually tried switching to a locale? If so, how did you do that,  and did it work for you?